app/components/table/component.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Story } from '@storybook/react/types-6-0';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import STAR_SVG from 'svgs/ui/star.svg?sprite';
+import Table, { TableProps, TableRow, LabelValue } from './component';
+
+export default {
+  title: 'Components/Table',
+  component: Table,
+  argTypes: {},
+};
+
+const Template: Story<TableProps> = ({ ...args }: TableProps) => (
+  <Table {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  headers: [
+    {
+      label: 'Best',
+      id: 'best',
+      customCell: (row: LabelValue) => {
+        if (row.value) return <Icon className="w-3 h-3" icon={STAR_SVG} />;
+        return '';
+      },
+    },
+    {
+      label: 'RUN',
+      id: 'run',
+    },
+    {
+      label: 'Score',
+      id: 'score',
+    },
+    {
+      label: 'Cost',
+      id: 'cost',
+    },
+    {
+      label: 'View on map',
+      id: 'view-on-map',
+      customCell: function selectSolutionButton({ isSelected }: TableRow) {
```

Comment:
Why are you using named functions?

Reply:
You get this error otherwise: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md

Reply:
Can't you use arrow function?